charts/cloudserver/templates/manager-deployment.yaml

```diff
@@ -39,10 +39,10 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
           args: ['npm', 'run', 'start_s3server']
-          livenessProbe:
```

Comment:
Interesting... I don't see why the non managers will be involved since they are not getting requests at all AFAIK. But these setup needs to be completely refactored. Thanks 👍